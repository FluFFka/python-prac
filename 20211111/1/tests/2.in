a = OrderedComplexMul(1.123456)
b = OrderedComplexMul(3+3.1j)
c = OrderedComplexMul(1.j,9.8j+555)
d = OrderedComplexMul(OrderedComplex(123,45.j+1j-123), c@b)
p = [a, b, c, d]
for i in range(4):
    for j in range(4):
        print(p[i]@p[j], p[i] < p[j])
