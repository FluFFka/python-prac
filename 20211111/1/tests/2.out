1.262153383936 False
3.370368 True
-11.009868800000001 False
86.506112 True
3.370368 False
18.61 False
1694.2 False
5101.72 True
-11.009868800000001 False
1694.2 False
309232.04 False
872832.6000000001 True
86.506112 False
5101.72 False
872832.6000000001 False
2474598.44 False
